documentationConfig.json

```diff
@@ -8,12 +8,7 @@
   },
   "plugins": ["node_modules/better-docs/category"],
   "source": {
-    "include": [
```

Comment:
Because the Plugin had Problems with our  in-line SwaggerUi Documentation. Furthermore, those are residues that Lukas had not changed after adapting the folder structure. Paths like `./../server/controllers` don't directly exist anymore and when further inspecting the files that had been there, they are now documented using swagger and not JSDoc. I've added the `testDatabaseHandler `and the `socketHelperFunctions`, but apart from that no files has the JSDoc documentation style